src/preprocessing.jl

```diff
@@ -36,29 +36,17 @@ function mk_regex(regex_string)
 end
 
 
-"""
-    remove_corrupt_utf8(str)
```

Comment:
I guess something, went wrong while resolving conflicts. All the Docstrings (offline documentation) from `preprocessing.jl` got removed.

Reply:
Yes, I `preprocessing.jl` while merging there was some problem.

Reply:
Cool. Thanks a lot.